pkg/asset-manager-utils/test/RebalancingRelayer.test.ts

```diff
@@ -120,18 +122,12 @@ describe('RebalancingRelayer', function () {
           it('rebalances the pool', async () => {
             const receipt = await relayer.connect(sender).joinPool(poolId, recipient.address, request);
 
-            expectEvent.inIndirectReceipt(await receipt.wait(), assetManagers[0].interface, 'Rebalanced', {
+            expectEvent.inIndirectReceipt(await receipt.wait(), assetManagers[0].interface, 'Rebalance', {
               poolId,
-              assetManager: assetManagers[0].address,
-              token: tokens.first.address,
-              force: false,
```

Comment:
Why remove these?

Reply:
I've replaced the MockAssetManager with a version which allows pulling cash into the managed balance and the events in these two contracts differ.